lib/barcodes_panel.dart

```diff
@@ -36,17 +37,19 @@ class BarcodesPanelState extends State<BarcodesPanel> {
     final barcodeType = _selectedBarcodeType;
 
     setState(() {
-      _descriptionError =
-          description.isEmpty ? "Opis nie może być pusty" : null;
-      _dataError = data.isEmpty ? "Kod kreskowy nie może być pusty" : null;
+      _descriptionError = description.isEmpty
+          ? Translations.of(context).error.emptyDescription
```

Comment:
Zamiast robić co chwile `Translations.of(context)` wynieś to do zmiennej na początku metody.
`Translations translations = Translations.of(context)` i wtedy będzie mógł używać tego tak `translations.error.emptyDescription`.